samples/Common.c

```diff
@@ -679,20 +679,32 @@ STATUS lookForSslCert(PSampleConfiguration* ppSampleConfiguration)
     return retStatus;
 }
 
-STATUS createSampleConfiguration(PCHAR channelName, SIGNALING_CHANNEL_ROLE_TYPE roleType, BOOL trickleIce, BOOL useTurn,
+STATUS createSampleConfiguration(PCHAR channelName, SIGNALING_CHANNEL_ROLE_TYPE roleType, BOOL trickleIce, BOOL useTurn, BOOL useIot,
```

Comment:
going to change useIoT to also use a new statically defined global

Reply:
Noticed the team modifying this sample, so left it to the team. thanks.